Application/EdFi.Ods.Api/Security/Authorization/Repositories/AggregateRootCriteriaProviderDecoratorBase.cs

```diff
@@ -99,10 +99,29 @@ public ICriteria GetCriteriaQuery(TEntity specification, IQueryParameters queryP
 
                     isSubjectNameAuthorizable = true;
 
+                    var filtersBackedByNewAuthViews = new List<string>
+                    {
+                        "LocalEducationAgencyIdToStudentUSI",
+                        "SchoolIdToStudentUSI"
+                    };
+
                     // Invoke the filter applicators against the current query
                     foreach (var applicator in applicators)
                     {
-                        var parameterValues = new Dictionary<string, object> { { filterDetails.ClaimEndpointName, filterDetails.ClaimValues } };
+                        Dictionary<string, object> parameterValues;
+
+                        if (filtersBackedByNewAuthViews.Contains(filterDetails.FilterName, StringComparer.OrdinalIgnoreCase))
+                        {
+                            parameterValues =
+                                new Dictionary<string, object> {{"SourceEducationOrganizationId", filterDetails.ClaimValues}};
+                        }
```

Comment:
This is the other half of the parameter renaming, where you're forcing the parameter name to be SourceEducationOrganizationId for student views.

Reply:
As discussed ,  I am not doing this change due to below reason 

if (!parameters.TryGetValue(filterPropertyName, out value))
                throw new Exception($"Unable to find parameter for filtering '{filterPropertyName}' on view '{viewName}'.");
Here we are checking filterPropertyName with parameters key "LocalEducationAgencyId"
So  I need to pass "LocalEducationAgencyId" for filterPropertyName instead of SourceEducationOrganizationId